deploy/kube-bench/templates/daemonset.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
```

Comment:
we should include the chart labels here (`kube-bench.labels` helper). look at the generated templates when you run `helm create` for reference how it should look like.